src/lcd.cpp

```diff
@@ -619,30 +619,77 @@ void LcdTask::displayInfoLine(LcdInfoLine line, unsigned long &nextUpdate)
   }
 }
 
-void LcdTask::displayScaledNumberValue(int line, const char *name, double value, int precision, const char *unit)
-{
-  static const char *mod[] = {
-    "",
-    "k",
-    "m",
-    "g",
-    "t",
-    "p"
-  };
-
-  int index = 0;
-  while (value > 1000 && index < ARRAY_ITEMS(mod))
+// Adjust the value down until its in the range 0-1000,
+// and return the SI prefix for the scaled number. 
+const char *LcdTask::ScaleNumberSI(double *value) {
   {
-    value /= 1000;
-    index++;
+    static const char *mod[] = {
+      "",
+      "k",
+      "m",
+      "g",
+      "t",
+      "p"
```

Comment:
I opened another PR before seeing this one to correct the capitalization of these units - might be worth updating it here since you're moving these around? https://github.com/OpenEVSE/ESP32_WiFi_V4.x/pull/344

Reply:
Not a fan of multiple changes per PR but #344 will conflict with this PR and visa-versa so ok if @rbsexton wants to add the unit update

Reply:
This one has been laying around!   I'd forgotten all about it.      I'll update this so that it can get pulled.

Reply:
Ok.   I'm pretty sure that the merge of PR#344 made mine unmergable.  I'm going to start over and I'll fix the rounding error while I'm at it.